### PR TITLE
Add EmptyPath for FchownatFlags

### PR DESCRIPTION
**Pull request by tathanhdinh**
Hello,

This PR is probably a fix for #1029. Thanks for any response.

### CODE REVIEW

**Comment by otavio** [comment]
Couldn't same thing to be done using `fchown()`?

**Comment by tathanhdinh** [comment]
@otavio thank you for the feedback but IMHO `fchown` doesn't need flags.

**Comment by otavio** [comment]
@tathanhdinh you got me wrong. I meant if it is worth adding this or using the `fchown` as an alternative.

**Comment by tathanhdinh** [comment]
Ah, sorry.
I don't know whether `fchown` can always be used. Does it always follow symbolic link?

**Comment by otavio** [comment]
The `fchown` operates on the file description and in this case, seems equivalent to `fchownat` with `EmptyPath` flag.

File: src/unistd.rs [hunk 1]
```diff
@@ -631,6 +631,8 @@ pub fn chown<P: ?Sized + NixPath>(path: &P, owner: Option<Uid>, group: Option<Gi
 pub enum FchownatFlags {
     FollowSymlink,
     NoFollowSymlink,
+    #[cfg(target_os = "linux")]
+    EmptyPath,
```
Comment (SteveLauC) [inline src/unistd.rs:635]: If we add this flag, then this type should no longer be an enum but a bitflag cause you can do `AT_EMPTY_PATH` and ` AT_SYMLINK_NOFOLLOW` at the same time

**Pull request closed without merging**